scripts/apps/search/directives/SearchResults.ts

```diff
@@ -171,6 +171,11 @@ export function SearchResults(
              * Schedule an update if it's not there yet
              */
             function queryItems(event?, data?) {
+                if (isObjectId(scope.search.repo.search)) {
```

Comment:
I hope this works for the initial query.

Reply:
we don't use it for initial query